### PR TITLE
a11y setup with errors ignored

### PR DESCRIPTION
**Pull request by jamesbhobbs**
* [x] Fixes https://github.com/bbc/simorgh/issues/305
~~* [ ] Tests added for new features~~

* [ ] Test engineer approval

### CODE REVIEW

**Comment by greenc05** [comment]
@benjaminhobbs do we plan to use https://github.com/bbc/a11y-tests-web ? The awesome @andymsuk will no doubt be happy to provide guidance if you are having trouble

File: package.json [hunk 1]
```diff
@@ -4,21 +4,24 @@
...
-    "lighthouse": "lighthouse http://localhost:7080/article/id --output-path=./reports/report.html --view",
+    "lighthouse": "lighthouse http://localhost:7080/article/scenario-01 --output-path=./reports/report.html --view",
     "postmerge": "./scripts/dependencyUpdateCheck.sh",
     "precommit": "npm run test:lint",
     "prepush": "./scripts/upToDateWithLatest.sh && npm run test:unit",
     "speculate": "speculate",
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test": "npm run test:lint && npm run test:unit && npm run data-validate",
+    "test:accessibility": "(lsof -t -i:7080 | xargs kill) && npm run build && run-p --race start bbc-a11y",
```
Comment (jtart) [inline package.json:23]: Not in this PR, but we should move `(lsof -t -i:7080 | xargs kill)` to its own script - we are using it in multiple places now.

File: a11y.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const pageWidths = [320, 400, 600, 1008, 1280];
```
Comment (jtart) [inline a11y.js:1]: Can this file be moved out of the root directory?

**Review by jtart — CHANGES_REQUESTED** [review]
Is there any reason to not use https://github.com/bbc/a11y-tests-web?

File: a11y.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const skipTests = [
+  'Design: Content resizing: Text must be styled with units that are resizable in all browsers',
+  'Structure: Containers and landmarks: Exactly one main landmark',
+];
+const urls = [
+  'http://localhost:7080/article/scenario-25',
```
Comment (jtart) [inline a11y.js:7]: We need to bear in mind that eventually these can't be hard-coded to localhost.

File: a11y.js (hunk 2 above)
Comment (jamesbhobbs) [reply a11y.js:1]: Not according to the docs of bbca11y. 

Also re 'Is there any reason to not use https://github.com/bbc/a11y-tests-web?' 1. the issue says to use this, but I evaluated it anyway, see https://github.com/bbc/simorgh/issues/331 and I've opened PRs on that repo to clarify that docker isn't needed. Aside from extending the functionality the setup is unclear and I wanted to tackle using it along with setup in jenkins so I can contribute back.

**Review by jtart — DISMISSED** [review]
Just need to resolve conflicts.

**Pull request merged**